### ISSUE TITLE
unable to get working with pinboard RSS feed

### ISSUE THREAD

**Issue by njm2112** [nhoizey/PinboardInGoogle#13]
Hoping someone who's successfully using this can provide some advice. Though the repo's readme says that the script does not currently work with Chrome (58.0.3029.110), it appears that it *would* work if only I could point the script to my Pinboard RSS feed. When running a Google search, the results page shows a space for Pinboard results at the top and provides the instructions for setup; however, when opening my pinboard.in page and clicking the RSS button, Chrome throws this error:

> This XML file does not appear to have any style information associated with it. The document tree is shown below.

Hoping that the readme is just out of date and that someone could point me in the right direction to get the script working!

Thanks.

**Comment by nhoizey** [comment]
The XML file is the Atom feed of your bookmarks.

Unfortunately, the script doesn't seem to work in Chrome, even with Tampermonkey, because I use the `GM_setValue` function from Greasemonkey.

**Comment by nhoizey** [comment]
I've just released v2.0 of the script, which now works also in Chrome, at least with the Tampermonkey extension.

It is a little less magic for configuration, because you need to edit the script manually.

**Comment by njm2112** [comment]
Hi, again -- sorry if this is a simple problem to solve but I am not at all experienced with javascript and am sort of learning as I go...

Using v2.0 of your script with Tampermonkey and Chrome 64.0.3282.140.

Seems that Tampermonkey's syntax checker is having two issues:

First, syntax and handling multiline strings. `Bad escaping of EOL. Use option multistr if needed` at this section of the script:
```
    // Add some style to it
    $('head').append('\
      <style>\
      #PinboardInGoogle { background-color: #eee; padding: .2em; margin-bottom: 1em; }\
      #PinboardInGoogle p { margin: .5em 0; }\
      #PinboardInGoogle p.title { color: #aaa; padding-left: 20px; background: url(https://pinboard.in/bluepin.gif) left center no-repeat; }\
      #PinboardInGoogle .how, #PinboardInGoogle .reset { float: right; font-size: .8em; }\
      #PinboardInGoogle.small li:nth-child(n+4) { display: none; }\
      </style>'
```

I tried using `` ` `` instead of `'` and though that cleared the `Bad escaping...` warning, the script still doesn't seem to work. Separately I also tried getting around the `Bad escaping...` warning by defining a variable to hold the `<style>` code as a single-line string substituting `;;` for the `\`, and then using RegEx via `newString.replace` to replace all of my `;;` with `\n`. No luck.

The second thing is that there's an expected `;` at the end of `` htmlLinks += \ `<a href="https://pinboard.in/t:${tag}" target="_blank">${tag}</a> ` ``, which I've added, but that doesn't seem to resolve the problem with the script functioning because of the initial warning above.

Would really like to be able to use this script so any help would be appreciated!

Thanks.
